### PR TITLE
Fix so that udevadm can be run from inside kubernetes container on the host

### PR DESCRIPTION
**Pull request by davidz627**
Fixes: #69

/assign @msau42

### CODE REVIEW

File: deploy/kubernetes/dev/node.yaml [hunk 1]
```diff
@@ -49,8 +49,17 @@ spec:
               mountPropagation: "Bidirectional"
             - name: plugin-dir
               mountPath: /csi
+            # The following mounts are required to trigger host udevadm from container
             - name: device-dir
-              mountPath: /host/dev
+              mountPath: /dev
```
Comment (msau42) [inline deploy/kubernetes/dev/node.yaml]: Can you add a comment on what each of these are needed for?

Comment (msau42) [reply deploy/kubernetes/dev/node.yaml]: Can all of these be readonly?

**Comment by msau42** [comment]
let me know when tests have been done (current e2es don't test the driver in a container or kubernetes)

**Comment by davidz627** [comment]
@msau42 now running Kubernetes integration test and it is passing :+1:

**Pull request merged**